benchmarks/transport_montecarlo_opacities.py

```diff
@@ -89,3 +89,16 @@ def time_pair_creation_opacity_calculation(
         calculate_opacity.pair_creation_opacity_calculation(
             energy, ejecta_density, iron_group_fraction
         )
+
+    @parameterize(
+        {
+            "Energy": [
+                511.0, 
+                255.5, 
+                0.0, 
+                511.0e7,
+            ]
+        }
+    )
+    def time_kappa_calculation(self, energy):
+        calculate_opacity.kappa_calculation(energy)
```

Comment:
Empty line at end of file

Reply:
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Is there any vs code extension to automate this task?